autoload/lsp/outline.vim

```diff
@@ -259,7 +259,7 @@ export def OpenOutlineWindow()
     highlight default link LSPTitle Title
   endif
 
-  prop_type_add('LspOutlineHighlight', {bufnr: bufnr(), highlight: 'Search'})
+  prop_type_add('LspOutlineHighlight', {bufnr: bufnr(), highlight: 'diffchange', override: true})
```

Comment:
Can you remove the 'highlight' group change here also?

Reply:
this here probably common choice, e.g tagbar is using this (diffchange) also.
but anyway, i follow if you preferred.